src/create/aws/create-http-route/create-route/02-setup-request.js

```diff
@@ -56,13 +57,14 @@ module.exports = function _02setupRequest(params, callback) {
             integrationHttpMethod: 'POST',
             uri,
             requestTemplates: {
-              'text/html': vtl,
-              'application/x-www-form-urlencoded': vtlForm,
-              'multipart/form-data': vtl,
               'application/json': vtl,
+              'application/octet-stream': vtlBinary,
```

Comment:
At some point you may also want to add other known binary content types. You’re going to want the same thing for pretty much every audio, video and image (except SVG) mime type.

Reply:
Right! Depends on how we want to approach that. On the one hand, we could tell people to just upload with `application/octet-stream` or encode with `multipart/form-data`.

Could also add templates for every common binary audio, video, and image format, which feels noisy. Sadly, we can't (yet) do something like `image/*`.